### ISSUE TITLE
Sessions saved in RaySession 0.5.4 won't fully re-open

### ISSUE THREAD

**Issue by sub26nico** [Houston4444/RaySession#11]
A session made with Ray Session 0.5.2 is successfully opened in ray session 0.5.4. 
But if I save and quit, some applications doesn't work when I relaunch the session. 
On LibraZiK2, session with seq24, Catia, Hydrogen, ZynAddSubFx and Jack Patch. 
At launch : 
```
[ray-daemon]Registered with GUI
[ray-daemon]osc.udp://debian:16187/
[ray-daemon]Attempting to open /home/sub26nico/Ray Sessions/test raysession0.5.4
[ray-daemon]Commanding unneeded and dumb clients to quit
[ray-daemon]Commanding smart clients to switch
[ray-daemon]Process has pid: 6679
[ray-daemon]The client "JACK Patch Memory" at "osc.udp://127.0.0.1:15805/" informs us it's ready to receive commands.
[ray-daemon]Client sends clean

[ZynAddSubFX-zynaddsub]

ZynAddSubFX - Copyright (c) 2002-2013 Nasca Octavian Paul and others
[ray-daemon]Process has pid: 6693
[ray-daemon]The client "ZynAddSubFX" at "osc.udp://127.0.0.1:19199/" informs us it's ready to receive commands.

[ZynAddSubFX-zynaddsub]
[INFO] exec-after-init
[INFO] startup OSC
[INFO] UI calbacks
[INFO] OSC replay
[INFO] auto_save setup
[INFO] NSM Stuff
[INFO] LASH Stuff
[INFO] Launching Zyn-Fusion...
[INFO] Main Loop...
Deactivating and closing JACK client

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ZynAddSubFX-zynaddsub]
Jack buffer resized

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ZynAddSubFX-zynaddsub]
Audio Started
Starting MIDI: ALSA

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ZynAddSubFX-zynaddsub]
Info, alsa midi port connected

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ray-daemon]Process has pid: 6694
[ray-daemon]The client "Ray Proxy" at "osc.udp://127.0.0.1:15806/" informs us it's ready to receive commands.
[ray-daemon]Client sends gui hidden

[ZynAddSubFX-zynaddsub]
Time for a fast load is 8.145ms load(0.011) class(0.001) spawn(8.133)...

[catia-catia]
PatchCanvas::splitGroup(1) - group is already splitted

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ray-daemon]Process has pid: 6680
[ray-daemon]The client "Hydrogen" at "osc.udp://127.0.0.1:10988/" informs us it's ready to receive commands.

[Hydrogen-hydrogen]
NSM: Successfully registered. NSM says: Howdy, what took you so long?
[ray-daemon]Telling all clients that session is loaded...
[ray-daemon]Telling client JACK Patch Memory that session is loaded.
[ray-daemon]Telling client Hydrogen that session is loaded.
[ray-daemon]Telling client ZynAddSubFX that session is loaded.
[ray-daemon]Telling client Ray Proxy that session is loaded.
[ray-daemon]Loaded
[ray-daemon]Done

[Hydrogen-hydrogen]
QObject::connect: No such signal LCDCombo::valueChanged( QString ) in /build/hydrogen-0.9.99~1.0.0-beta1+20180409~gitcf53be9/src/gui/src/InstrumentEditor/InstrumentEditor.cpp:476

[ray-daemon]Telling JACK Patch Memory to save
[ray-daemon]Telling Hydrogen to save
[ray-daemon]Telling ZynAddSubFX to save
[ray-daemon]Telling Ray Proxy to save
[ray-daemon]Client sends clean

[ZynAddSubFX-zynaddsub]
Unknown address<BACKEND:online> '/vu-meter:ffffffffffffffffffffff'
Unknown path '/vu-meter:ffffffffffffffffffffff'
Unknown address<BACKEND:online> '/broadcast:'
Unknown path '/broadcast:'
Unknown address<BACKEND:online> '/active_keys:FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
Unknown path '/active_keys:FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
Unknown address<BACKEND:online> '/save_xmz:'
Unknown path '/save_xmz:'

[ray-daemon]Session saved.
[ray-daemon]Done.

```
After save : 
```
[ray-daemon]Registered with GUI
[ray-daemon]osc.udp://debian:16187/
[ray-daemon]Attempting to open /home/sub26nico/Ray Sessions/test raysession0.5.4
[ray-daemon]Commanding unneeded and dumb clients to quit
[ray-daemon]Commanding smart clients to switch
[ray-daemon]Process has pid: 6789
[ray-daemon]The client "JACK Patch Memory" at "osc.udp://127.0.0.1:16265/" informs us it's ready to receive commands.
[ray-daemon]Client sends clean

[ZynAddSubFX-zynaddsub]

ZynAddSubFX - Copyright (c) 2002-2013 Nasca Octavian Paul and others
[ray-daemon]Process has pid: 6803
[ray-daemon]The client "ZynAddSubFX" at "osc.udp://127.0.0.1:19660/" informs us it's ready to receive commands.

[ZynAddSubFX-zynaddsub]
[INFO] exec-after-init
[INFO] startup OSC
[INFO] UI calbacks
[INFO] OSC replay
[INFO] auto_save setup
[INFO] NSM Stuff
[INFO] LASH Stuff
[INFO] Launching Zyn-Fusion...
[INFO] Main Loop...
Deactivating and closing JACK client

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ray-daemon]Process has pid: 6804
[ray-daemon]The client "Ray Proxy" at "osc.udp://127.0.0.1:16266/" informs us it's ready to receive commands.

[ZynAddSubFX-zynaddsub]
Starting Audio: JACK

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ray-daemon]Client sends gui hidden

[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ZynAddSubFX-zynaddsub]
Jack buffer resized
[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[catia-catia]
PatchCanvas::splitGroup(1) - group is already splitted

[ZynAddSubFX-zynaddsub]
Info, alsa midi port connected

[catia-catia]
PatchCanvas::splitGroup(4) - group is already splitted

[ZynAddSubFX-zynaddsub]
Time for a fast load is 3.469ms load(0.009) class(0.001) spawn(3.459)...
[Hydrogen-hydrogen]
convolution: samplerate mismatch preset:48000 host:44100

[ray-daemon]Process has pid: 6790
[ray-daemon]The client "Hydrogen" at "osc.udp://127.0.0.1:10351/" informs us it's ready to receive commands.

[Hydrogen-hydrogen]
NSM: Successfully registered. NSM says: Howdy, what took you so long?
[ray-daemon]Telling all clients that session is loaded...
[ray-daemon]Telling client JACK Patch Memory that session is loaded.
[ray-daemon]Telling client Hydrogen that session is loaded.
[ray-daemon]Telling client ZynAddSubFX that session is loaded.
[ray-daemon]Loaded
[ray-daemon]Done

[Hydrogen-hydrogen]
QObject::connect: No such signal LCDCombo::valueChanged( QString ) in /build/hydrogen-0.9.99~1.0.0-beta1+20180409~gitcf53be9/src/gui/src/InstrumentEditor/InstrumentEditor.cpp:476

[ray-daemon]Process has pid: 6835
[ray-daemon]The client "Ray Proxy" at "osc.udp://127.0.0.1:16276/" informs us it's ready to receive commands.
[ray-daemon]Client sends gui hidden
[ray-daemon]Process has pid: 6843
[ray-daemon]The client "Ray Proxy" at "osc.udp://127.0.0.1:16278/" informs us it's ready to receive commands.
[ray-daemon]Client sends gui hidden
```

Seq24 doesn't start, and if I try to start it with the button on the RS interface, it doesn't work.

**Comment by sub26nico** [comment]
If I downgrade to raysession 0.5.2, launch and save the session, then upgrade to 
ray session 0.5.4, the session works perfectly (seq24 works). If I save, same issue.

**Comment by sub26nico** [comment]
Same issue with CalfJackHost and jackMixer

**Comment by sub26nico** [comment]
So, I create a session in RaySession 0.5.2 with CalfJackHost, JackMixer, Seq24, ZynAddSubFx, 
Hydrogen and Catia. I save the session and reload it. It works, everything is launched. 
Here the 'raysession.xml : 
```
<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE RAYSESSION>
<RAYSESSION name="SessionTestDebug" VERSION="0.5.2">
 <Clients>
  <client launched="1" prefix_mode="2" executable="ray-jackpatch" name="JACK Patch Memory" id="patch" icon="curve-connector"/>
  <client launched="1" prefix_mode="2" executable="ray-proxy" name="Ray Proxy" id="calf" icon="calf" label="Calf JACK Host"/>
  <client launched="1" prefix_mode="2" executable="ray-proxy" name="Ray Proxy" id="mixer" icon="jack_mixer" label="Jack Mixer"/>
  <client launched="1" prefix_mode="2" executable="ray-proxy" name="Ray Proxy" id="seq24" icon="seq24" label="Seq24"/>
  <client launched="1" prefix_mode="2" executable="zynaddsubfx" name="ZynAddSubFX" id="zynaddsub" icon="zynaddsubfx"/>
  <client launched="1" prefix_mode="2" executable="hydrogen" name="Hydrogen" id="hydrogen" icon="h2-icon" label="Hydrogen"/>
  <client launched="1" prefix_mode="2" executable="catia" name="catia" id="catia" icon="catia"/>
 </Clients>
 <RemovedClients/>
</RAYSESSION>
```

**Comment by sub26nico** [comment]
Then I upgrade RaySession to 0.5.4 version. I load the session, it works. 
I save the session : only Catia, ZynAdd and Hydrogen are launched. Others applications won't open, 
and stay gray. 
here the 'raysession.xml' : 
```
<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE RAYSESSION>
<RAYSESSION VERSION="0.5.4" name="SessionTestDebug">
 <Clients>
  <client launched="1" icon="curve-connector" id="patch" executable="ray-jackpatch" name="JACK Patch Memory"/>
  <client launched="1" icon="calf" label="Calf JACK Host" id="calf" executable="ray-proxy" name="Ray Proxy" gui_visible="0"/>
  <client launched="1" icon="jack_mixer" label="Jack Mixer" id="mixer" executable="ray-proxy" name="Ray Proxy" gui_visible="0"/>
  <client launched="1" icon="seq24" label="Seq24" id="seq24" executable="ray-proxy" name="Ray Proxy" gui_visible="0"/>
  <client launched="1" icon="zynaddsubfx" id="zynaddsub" executable="zynaddsubfx" name="ZynAddSubFX"/>
  <client launched="1" icon="h2-icon" label="Hydrogen" id="hydrogen" executable="hydrogen" name="Hydrogen"/>
  <client launched="1" icon="catia" id="catia" executable="catia" name="catia"/>
 </Clients>
 <RemovedClients/>
</RAYSESSION>
```

**Comment by sub26nico** [comment]
Seems to be fixed with the 0.5.5 version.

**Comment by sub26nico** [comment]
After testing version 0.5.5, no more problems, I close here.
